mph/tools.py

```diff
@@ -13,7 +13,7 @@
 # Dependencies                         #
 ########################################
 from numpy import array                # numerical arrays
-
+from .model import Model

```

Comment:
I understand how this would be convenient, but my intention was to keep the `tools` module independent of the rest of the code base.

Reply:
Removed that from the branch